src/plugins/phoenix.js

```diff
@@ -22,6 +22,41 @@ export default {
         },
         // FIXME: optional publicContext parameter is a mess
         downloadFile(file, publicContext = null) {
+          if (publicContext === null) {
+            const url = this.$client.helpers._webdavUrl + file.path
+
+            return this.$client.signUrl(url).then(signedUrl => {
+              fetch(signedUrl, {
+                method: 'HEAD',
```

Comment:
on the server side the signed url will still be checked, but since you are using HEAD it will never match because the signed URL includes the method (GET). Why even chech at all? if it is a 404 the browser will tell you that the file does not exist ... why do you want to double check that?

Reply:
please check the most recent solution